inst/stan/simulate_infections.stan

```diff
@@ -23,7 +23,23 @@ data {
 }
 
 transformed data {
-  int delay_max_total = sum(delay_max) - num_elements(delay_max) + 1;
+  int gt_max_fixed =
+    num_elements(gt_np_pmf) - num_elements(gt_np_pmf_groups) + 1;
+  int gt_max_total = gt_max_fixed + sum(gt_max) - num_elements(gt_max);
+  int delay_max_fixed =
```

Comment:
arrrrrhhh code duplication (agree hard to avoid)

Reply:
I have no idea if chunks are only compiled once. I assume not but if they are moving to chunks could drastically reduce compile times

Reply:
no, from looking at `src` after compilation I think unfortunately these are not compiled into separate object but instead it's separate compilation per model in `inst/stan`